README.md

```diff
@@ -51,6 +51,98 @@ function handleSubmit(event) {
 </template>
 ```
 
+## Non-primitive values
+
+In some cases, you might want to select values that aren't primitive types, e.g selecting a user from a list of users.
+Unfortunately, FormData only supports strings. To work around this, `form-data-utils` provides some utility functions
+to "attach" a non-primitive value to an element:
+
+- `setValue(element: HTMLElement, value: unknown)` - binds a given value to a given element (value can be anything, e.g objects, arrays, etc)
+- `deleteValue(element: HTMLElement)` - undoes a previous `setValue` call
+
+After setting a value with `setValue`, that value will be considered by the `dataFrom` function.
+
+> [!NOTE]  
+> The ideal way to call these functions will probably vary with the framework you're using.
+
+For example, in [Ember.js](https://emberjs.com/), you would use these functions in a modifier that you can then apply to elements.
+
+```gjs
+import { dataFrom, deleteValue as formDeleteValue,setValue as formSetValue } from 'form-data-utils';
```

Comment:
instead of renaming these, for demo purposes (keeping it small), let's rename the modifier to `associateObject` or something to that effect?

Reply:
Sounds good to me, but I find that naming the modifier as `setValue` very natural, as this is intended as a "replacement" of `value="something"`.

Perhaps we could rename the imports to something else?

I'm find either way. 👍

Reply:
hm, maybe then?:
```js
import * as utils from 'form-data-utils';

utils.dataFrom(...)
utils.setValue(...)
```
?